examples/hybrid/driver.jl

```diff
@@ -235,7 +235,7 @@ integrator = OrdinaryDiffEq.init(
     dt = dt,
     adaptive = false,
     progress = show_progress_bar,
-    progress_steps = 1000,
+    progress_steps = 1,
```

Comment:
This makes the output really bloated, can compromise at 100?

Reply:
Since `show_progress_bar = is interactive()`, this only has an effect when running from the REPL. So having this set to 1 improves responsiveness without impacting CI.

Reply:
I think we should keep progress bars in CI, just not this frequently

Reply:
I'm confused as to what benefit we get from having a progress bar in CI? It just garbles the output (since the progress bar is meant for displaying in a terminal), and I don't think many people sit around watching CI logs as their code runs.

Reply:
It's useful for long runs.

Reply:
`progress_steps = isinteractive() ? 1 : 1000`?

Reply:
That's fine by me